jenkins/clouds/azure/daily/tagging/Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+def accounts_list = ['perfscale']
+pipeline {
+    agent {
+        docker {
+            label 'cloud-governance-worker'
+            image 'quay.io/cloud-governance/centos-stream8-podman:latest'
+            args  '-u root -v /etc/postfix/main.cf:/etc/postfix/main.cf --privileged'
+        }
+    }
+    environment {
+        QUAY_CLOUD_GOVERNANCE_REPOSITORY = credentials('QUAY_CLOUD_GOVERNANCE_REPOSITORY')
+        POLICIES_IN_ACTION = '[]'
+        AZURE_CLIENT_SECRET = credentials('cloud-governance-azure-client-secret')
+        AZURE_TENANT_ID = credentials('cloud-governance-azure-tenant-id')
+        AZURE_ACCOUNT_ID = credentials('cloud-governance-azure-account-id')
+        AZURE_CLIENT_ID = credentials('cloud-governance-azure-client-id')
+        ES_HOST = credentials('cloud-governance-es-host')
+        ES_PORT = credentials('cloud-governance-es-port')
+        LDAP_HOST_NAME = credentials('cloud-governance-ldap-host-name')
+        GLOBAL_TAGS = credentials('GLOBAL_TAGS')
+        contact2 = "yinsong@redhat.com"
+    }
+    stages {
+        stage('Checkout') { // Checkout (git clone ...) the projects repository
+          steps {
+            checkout scm
+           }
+        }
+        stage('Initial Cleanup') {
+            steps {
+                 sh '''if [[ "$(podman images -q ${QUAY_CLOUD_GOVERNANCE_REPOSITORY} 2> /dev/null)" != "" ]]; then podman rmi -f $(podman images -q ${QUAY_CLOUD_GOVERNANCE_REPOSITORY} 2> /dev/null); fi'''
+            }
+        }
+        stage('Run Azure Policies') {
+            steps {
+                 script {
+                    for (account in accounts_list ) {
+                        echo "Running for account ${account.toUpperCase()}"
+                        withCredentials([string(credentialsId: "${account}-azure-client-secret", variable: 'client_secret'),
+                                        string(credentialsId: "${account}-azure-client-id", variable: 'client_id'),
+                                        string(credentialsId: "${account}-azure-tenant-id", variable: 'tenant_id'),
+                                        string(credentialsId: "${account}-azure-subscription-id", variable: 'subscription_id'),
+                                        string(credentialsId: "${account}-azure-account-id", variable: 'account_id')]) {
```

Comment:
Have you verified the secrets with above names/format existed in jenkins secrets.

Reply:
It must be there because I have copied it from [here](https://github.com/redhat-performance/cloud-governance/blob/1b6de2bc746f99352089b232c4a73bedf41a5b95/jenkins/clouds/azure/daily/policies/Jenkinsfile#L36-L44)

Reply:
Moved to [policy](https://github.com/redhat-performance/cloud-governance/tree/main/jenkins/clouds/azure/daily/policies) jenkins because need to run once a day because its tag all the account resource groups with global tags.